api/lib/source/polkadotV0-source.js

```diff
@@ -702,14 +703,21 @@ class polkadotAPI {
         )
         .concat(
           treasuryProposals.proposals.map(async (proposal) => {
-            // should work. treasury and council proposals are virtually the same object
+            const treasuryProposal = proposal.council[0]
+              ? proposal.council[0]
+              : undefined
```

Comment:
```suggestion
            const treasuryProposal = proposal.council[0]
```

Reply:
true! It doesn't make sense. But why cannot I commit this suggestion?

Reply:
looks like a github bug 🐛